crates/amaru-stores/src/rocksdb/mod.rs

```diff
@@ -159,8 +159,21 @@ impl RocksDB {
         self.db.transaction()
     }
 
+    fn get<T: for<'d> cbor::decode::Decode<'d, ()>>(
+        &self,
+        key: &str,
+    ) -> Result<Option<T>, StoreError> {
+        self.db
+            .get(key)
+            .map_err(|err| StoreError::Internal(err.into()))?
+            .map(|bytes| cbor::decode(&bytes))
+            .transpose()
+            .map_err(StoreError::Undecodable)
+    }
```

Comment:
Unnecessary refactor, which came from one of the iteration where the `get` logic was repeated across two calls. I ended up reverting most of that increment, but kept the refactor because it still somewhat _made_ sense.